app/components/UserLink/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { UnknownUser } from 'app/store/models/User';
+import { ProfilePicture } from '../Image';
+import { Link } from 'react-router-dom';
+import { Flex } from '@webkom/lego-bricks';
+
+type UserLinkProps = {
+  user: Pick<
+    UnknownUser,
+    'profilePicture' | 'username' | 'profilePicturePlaceholder' | 'fullName'
+  >;
+};
+
+const UserLink = ({ user }: UserLinkProps) => {
+  return (
+    <Flex alignItems="center" gap="var(--spacing-sm)">
```

Comment:
should these perhaps wrap?
```suggestion
    <Flex wrap alignItems="center" gap="var(--spacing-sm)">
```